treeherder/etl/taskcluster_pulse/parse_route.py

```diff
@@ -34,6 +34,8 @@ def parseRoute(route):
     if len(parsedRoute) == 5:
         pushId = parsedRoute[4]
 
-    pushInfo["pushId"] = pushId if int(pushId) else None
+    # XXX: Can you please evaluate I'm doing this right?
```

Comment:
Yeah, this is a little tricky to parse out.  But it looks right.  Though I fear using the term ``pushId`` here could be confusing since that means something different in Treeherder.  Could you use a better term here?